tutorials/sinon/check-arguments.md

```diff
@@ -0,0 +1,95 @@
+To check what arguments a stub was called with, you have several options as stubs are treated as spies in sinon.js.
```

Comment:
Saying that stubs are spies isn't useful because you haven't introduced spies yet, and the distinction is more likely to cause confusion than anything. Replace with "There are several ways to check what arguments a stub was called with in Sinon. Our recommended approaches are `calledWith()` and related helpers, and `getCall()`.